workflows/devnet-build/Jenkinsfile

```
@@ -31,7 +37,17 @@ pipeline {
                         }
                         dir('build') {
                             ansiColor('xterm') {
-                                sh 'make LISK_NETWORK=devnet'
+                                sh '''
+                                    VERSION=$( jq --raw-output '.version' ../package.json )
+                                    SHORT_HASH=$( git rev-parse --short HEAD )
+                                    OUTPUT_FILE="lisk-$VERSION-$SHORT_HASH-Linux-x86_64.tar.gz"
+                                    FILE_EXISTS=$( cat .s3_files |grep $OUTPUT_FILE| wc -l )
```

Useless use of `cat`.

```suggestion
                                    FILE_EXISTS=$( grep -c $OUTPUT_FILE .s3_files )
```